core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -622,8 +622,9 @@ portfolio.pending.step2_buyer.halCashInfo.msg=You need to send a text message wi
   trade ID ({0}) to the BTC seller.\nThe seller''s mobile nr. is {1}.\n\n\
   Did you send the code to the seller?
 portfolio.pending.step2_buyer.fasterPaymentsHolderNameInfo=Some banks might require the receiver's name. \
-  The UK sort code and account number is sufficient for a Faster Payment transfer and the receivers name is not verified \
-  by any of the banks.
+  The UK sort code and account number used to be sufficient for a Faster Payment transfer, but since then some banks \
+  have started verifying the receiver's name. For this reason, accounts created in old Bisq clients do not provide the \
+  trade partner with an owner full name. Please use trade chat to avoid any issues in such cases.
```

Comment:
```suggestion
  Faster Payment accounts created in old Bisq clients do not provide the receiver's name, \
  so please use trade chat to obtain it (if needed).
```

Reply:
OK, I'll apply that change. I think the text 

> Some banks might require the receiver's name ...

should possibly be changed to 

> Some banks might verify the receiver's name ...

with the suggested change, to emphasise that the issue is that they've started checking it. 

(Although my bank still accepts dummy names - I'm not sure how much longer, though.)